weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ScorersPage/ZodSchemaForm.tsx

```diff
@@ -698,23 +692,32 @@ const NumberField: React.FC<{
   const max =
     (unwrappedSchema._def.checks.find(check => check.kind === 'max') as any)
       ?.value ?? undefined;
+  const fieldDescription = getFieldDescription(fieldSchema);
 
   return (
-    <TextFieldWithLabel
-      label={keyName}
-      type="number"
-      value={(value ?? '').toString()}
-      onChange={newValue => {
-        const finalValue = newValue === '' ? undefined : Number(newValue);
-        if (
-          finalValue !== undefined &&
-          (finalValue < min || finalValue > max)
-        ) {
-          return;
-        }
-        updateConfig(targetPath, finalValue, config, setConfig);
-      }}
-    />
+    <Box display="flex" alignContent="center" justifyContent="space-between">
+      <TextFieldWithLabel
+        label={keyName}
+        type="number"
+        value={(value ?? '').toString()}
+        style={{width: '100%'}}
+        onChange={newValue => {
+          const finalValue = newValue === '' ? undefined : Number(newValue);
+          if (
+            finalValue !== undefined &&
+            (finalValue < min || finalValue > max)
+          ) {
+            return;
+          }
+          updateConfig(targetPath, finalValue, config, setConfig);
+        }}
+      />
+      {fieldDescription && (
+        <Box display="flex" alignItems="center" sx={{marginTop: '14px'}}>
+          <DescriptionTooltip description={fieldDescription} />
```

Comment:
this styling seems odd to me

Reply:
yeah because the icon gets centered in the whole div which is tall because of the TextFieldWithLabel being multi-line. We could add the description in TextFieldWithLabel but that breaks the pattern.